lib/ts/Arena.h

```diff
@@ -21,12 +21,13 @@
   limitations under the License.
  */
 
-#ifndef __ARENA_H__
-#define __ARENA_H__
+#pragma once
```

Comment:
I'm on board with this if all our supported platforms can use it (in the past they couldn't) but I think this needs a separate PR where it's done for every header file for consistency.